docs/rfcs/UserProfilesv2_schema.json

```diff
@@ -218,130 +286,44 @@
                 "metadata",
```

Comment:
Should this be instead
```
                 "metadata": { "$ref": "#/definitions/Metadata"},
```

Reply:
no, refs are for "object types", this is for which attribute is required